js/similar-ads.js

```diff
@@ -62,8 +62,17 @@ const createPopup = ({offer, author}) => {
   checkAvailableData(offer.features, adFeatures);
   checkAvailableData(offer.photos, adPhotos);
 
-  createFeatureElements(featuresList, offer.features, 'popup__feature--');
-  createPhotoElements(offer.photos, adPhotos);
+  if (typeof offer['features'] !== 'undefined') {
+    createFeatureElements(featuresList, offer.features, 'popup__feature--');
+  } else {
+    adFeatures.hidden = true;
+  }
+
+  if (typeof offer['photos'] !== 'undefined') {
```

Comment:
и тут
можно ещё так
```suggestion
  if (offer.photos) {
```